index.php

```diff
@@ -1,9 +1,16 @@
 <?php
 
 include_once('connection.php');
-include_once("src/controller/LoginController.php");
-include_once("src/controller/WelcomeController.php");
-include_once("src/controller/UserController.php");
+include_once("src/Controller/LoginController.php");
+include_once("src/Controller/WelcomeController.php");
+include_once("src/Controller/UserController.php");
+include_once("src/PasswordHasher/Md5Hasher.php");
+include_once("src/PasswordHasher/Sha1Hasher.php");
+include_once("src/PasswordHasher.php");
+include_once("src/Entity/User.php");
+
+$passwordHasher = new Sha1Hasher();
+$user = new User();
```

Comment:
Nelabai gera idėja User objektą sukurti jau čia, nes per index puslapį eina visi requestai, bet ne visiems jiems user objektas yra reikalingas ar naudojamas (pvz pas tave login psl kai vartotojas dar nežinomas, taip pat svetainei padidėjus atsiranda visokie kontaktų ar privatumo politikos psl kuriuose vertotojas gali būt neprisijungęs). Konkrečiai dabar ši problema index puslapyje yra ir su visais include_once (ne visur jų visų reikia) ir su password hasher instance'o sukūrimu. Bet šias problemas spręsim ateityje (kai ko papildomai pramokę ar sužinoję), o var User objekto kūrimą ir naudojimą tik tada kai to reikia galėtum padaryti jau dabar